utils/quizzes/lesson-4-quiz.json

```diff
@@ -0,0 +1,51 @@
+{
+  "title": "Quiz: Lesson 4",
+  "questions": [
+    {
+      "question": "Where do the values for event parameters get stored?",
+      "options": [
+        {
+          "answer": "In the block data"
+        },
+        {
+          "answer": "In transaction logs",
+          "correct": true
+        },
+        {
+          "answer": "They are not stored"
+        }
+      ]
+    },
+    {
+      "question": "Which of these properties does Ethereum have?",
```

Comment:
I wonder if one of the final questions should be pulled from the TokenURI and SVG sections -- namely discussions of strategies for breaking up the logic to make it more testable?

And maybe a question about TDD in the outro?

(these would replace this general Ethereum questions and the following "What can we use a smart contract for?" questions)

Reply:
My bad @briangershon. I should have emptied the .json files of all this text too (like I did with the mini-quiz questions last night). This the pre-fill text/questions from https://quiz-editor.vercel.app/ (you can always find this in /utils/quiz-creation-tools.txt)